modules/ROOT/pages/design-ghs-install-authorize.adoc

```diff
@@ -49,7 +49,7 @@ In the email settings for the GitHub account you want to use with GitHub Synchro
 
 === Install the GitHub Synchronization Application
 
-. Navigate to the https://github.com/apps/mulesoft-github-integration[GitHub Synchronization application page^].
+. Navigate to the https://github.com/apps/mulesoft-github-integration[GitHub Synchronization application page US] or https://github.com/apps/muesoft-github-integration-eu[Github Synchronization application page EU] depending on your region.
```

Comment:
Why is the region at the end? Isn't it "US GitHub Synchronization application page" or "EU GitHub Synchronization application page"?